### PR TITLE
Feature/mainpage speakers

### PR DESCRIPTION
**Pull request by pedromsfernandes**
Using a custom carousel component to show the speakers, allowing for infinite scrolling and therefore no blank space. Looking forward to your input regarding possible adjustments!

### CODE REVIEW

File: src/components/utils/carousel.module.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  border: 2px solid white;
+  background-color: var(--color1);
+  color: white;
+  cursor: pointer;
+  padding: 0.3rem 0.5rem;
+  border-radius: 50%;
+  outline: none;
+
+  opacity: 0.7;
```
Comment (smbea) [inline src/components/utils/carousel.module.css]:
```suggestion
  opacity: 0.4;
```

File: src/components/utils/carousel.module.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  transition: opacity 0.3s ease-in-out;
+}
+
+.carouselWrapper button:hover {
+  opacity: 1;
```
Comment (smbea) [inline src/components/utils/carousel.module.css]:
```suggestion
  opacity: 0.6;
```

**Comment by smbea** [comment]
> Regarding the UI, the only thing that I noticed was that the arrows in the buttons were not centered.

Oh you're right, hand't noticed. Can't unsee now. Please check that out if you can!

**Comment by pedromsfernandes** [comment]
Does it look better now? I'm still a bit confused but the box model looks good I think.

**Pull request merged**